src/laboratory/forms.py

```diff
@@ -863,3 +863,29 @@ class Meta:
             'name': genwidgets.TextInput,
             'description': genwidgets.Textarea
         }
+
+
+class MaterialForm(MaterialCapacityObjectForm, forms.ModelForm):
+
+    def __init__(self, *args, **kwargs):
+        modal_id = kwargs.pop('modal_id')
+        laboratory_pk = kwargs.pop('laboratory_pk')
```

Comment:
laboratory_pk variable is not necessary, just send required data. Also you can copy all fields in MaterialCapacityObjectForm form inside this other form because is a new form just for material crud, the MaterialCapacityObjectForm form will be deleted after this process.